index.css

```
@@ -4,10 +4,15 @@
 	--github-red: #cb2431;
 	--github-blue: #0366d6;
```

The contrast of the blue links is a little low, this should fix it

```suggestion
	--github-blue: #58a6ff;
```

Other colors should also be updated from GitHub, like the red ones